sale_order_line_no_product_creation/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2020 Mikel Arregi Etxaniz - AvanzOSC
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+{
+    "name": "Remove option to create product from sale order line form",
+    "version": "12.0.1.0.0",
+    "license": "AGPL-3",
+    "depends": [
+        "sale",
+    ],
+    "author": "AvanzOSC",
+    "website": "http://www.avanzosc.es",
+    "category": "",
+    "data": [
+         "views/sale_order_view.xml",
```

Comment:
sale_order_line_no_product_creation/__manifest__.py:14:10: E126 continuation line over-indented for hanging indent